### PR TITLE
Install unversioned engine plugin libs

### PR DESCRIPTION
**Pull request by iche033**
changes made by looking at [ign-physics](https://github.com/ignition-release/ign-physics2-release/blob/master/ubuntu/debian/libignition-physics-dartsim.install) as example

Signed-off-by: Ian Chen <ichen@osrfoundation.org>

### CODE REVIEW

**Review by nkoenig — APPROVED** [review]

File: ubuntu/debian/libignition-rendering-ogre1-dev.install [hunk 1]
```diff
@@ -3,4 +3,5 @@ usr/include/ignition/rendering[0-99]/ignition/rendering/ogre.hh
 usr/lib/*/libignition-rendering[0-99]-ogre.so
 usr/lib/*/pkgconfig/ignition-rendering[0-99]-ogre.pc
 usr/lib/*/cmake/ignition-rendering[0-99]-ogre/*
+usr/lib/*/ign-rendering-[0-99]/engine-plugins/libignition-rendering*-ogre*.so
```
Comment (chapulina) [inline ubuntu/debian/libignition-rendering-ogre1-dev.install]: This will install `ogre2` as well, right?

**Review by chapulina — CHANGES_REQUESTED** [review]

File: ubuntu/debian/libignition-rendering-ogre1-dev.install (hunk 1 above)
Comment (chapulina) [inline ubuntu/debian/libignition-rendering-ogre1-dev.install]:
```suggestion
usr/lib/*/ign-rendering-[0-99]/engine-plugins/libignition-rendering*-ogre.so
```

File: ubuntu/debian/libignition-rendering-ogre1.install [hunk 2]
```diff
@@ -1 +1,2 @@
 usr/lib/*/libignition-rendering[0-99]-ogre.so.*
+usr/lib/*/ign-rendering-[0-99]/engine-plugins/libignition-rendering*-ogre*.so.*
```
Comment (chapulina) [inline ubuntu/debian/libignition-rendering-ogre1.install]:
```suggestion
usr/lib/*/ign-rendering-[0-99]/engine-plugins/libignition-rendering*-ogre.so.*
```

**Review by chapulina — CHANGES_REQUESTED** [review]
I think this should work

File: ubuntu/debian/libignition-rendering-ogre1-dev.install (hunk 1 above)
Comment (iche033) [reply ubuntu/debian/libignition-rendering-ogre1-dev.install]: 65d975fe6dc090d495126884285e639de449fbb1

File: ubuntu/debian/libignition-rendering-ogre1.install (hunk 2 above)
Comment (iche033) [reply ubuntu/debian/libignition-rendering-ogre1.install]: 65d975fe6dc090d495126884285e639de449fbb1

**Review by chapulina — APPROVED** [review]
:ship: it!

File: ubuntu/debian/libignition-rendering-ogre1-dev.install (hunk 1 above)
Comment (iche033) [reply ubuntu/debian/libignition-rendering-ogre1-dev.install]: I dont see a ogre-dev.so  or files with other suffix strings so I think removing the asterisk works. 65d975fe6dc090d495126884285e639de449fbb1

**Pull request merged**